File: network/api/iss/iss.php
<?php

// API Request example in PHP

$req = file_get_contents('http://api.open-notify.org/iss-pass.json?lat=44.6134227&lon=1.1366893');
$api = json_decode($req);

if ($api->message == 'success') {

	foreach ($api->response as $issRes)
		        {
		$issDuration = round($issRes->duration / 60) . ' minutes';
		$issDate = date('Y-m-d H:i:s', $issRes->risetime);
		
		echo "
            	Duration: $issDuration \n
            	Date: $issDate\n
		==============================\n
		";
    }

}

File: network/api/iss/iss.py
# API Request example in Python

import urllib.request
import json
from datetime import datetime

req = urllib.request.Request('http://api.open-notify.org/iss-pass.json?lat=44.6134227&lon=1.1366893')
res = urllib.request.urlopen(req)
page = res.read()
api = json.loads(page)

if api['message'] == 'success':

    for issRes in api['response']:
        
        issDur = str(int(issRes['duration']) // 60) + ' minutes'
        issTime = datetime.fromtimestamp(issRes['risetime'])
        
        ret = '''
            Duration: {}\n
            Date: {}\n
            ==========================
        '''.format(issDur, issTime)
        
        print(ret)
